packages/parent-bridge/tests/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-ckb-tool = { git = "https://github.com/jjyr/ckb-tool.git", tag = "v0.1.1" }
-ckb-testtool = { git = "https://github.com/jjyr/ckb-tool.git", tag = "v0.1.1" }
+ckb-tool = { git = "https://github.com/jjyr/ckb-tool.git", branch="master"}
```

Comment:
@Strumster  Can we change this to use the official releases. Please note there may be some folder changes that would need to be changed for all contracts. Then test that nothing was broken.

https://github.com/nervosnetwork/capsule/releases

Reply:
load_header function calls don't work in official releases of dev tool, so audit delay wouldn't function with official version. Hopefully dev ads tag to the updated version.

Reply:
Sure can test as is for now.

Reply:
My bad i was confusing the libraries